libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -551,8 +570,12 @@ class AP_AHRS
         uint8_t fly_forward             : 1;    // 1 if we can assume the aircraft will be flying forward on its X axis
         uint8_t correct_centrifugal     : 1;    // 1 if we should correct for centrifugal forces (allows arducopter to turn this off when motors are disarmed)
         uint8_t wind_estimation         : 1;    // 1 if we should do wind estimation
+        uint8_t done_takeoff           : 1;    // 1 if vehicle has done a takeoff
```

Comment:
I prefer that we call it is_flying or in_flight. 'Done takeoff' could be true after the first takeoff and stay true, so is ambiguous.